Example/ViewController.m

```diff
@@ -175,12 +272,60 @@ - (void)getSessionData
                 [self.logVC logMessage:@"-------------Finished fetching session data!-------------"];
             }
         
-            self.navigationItem.leftBarButtonItem = nil;
-            self.title = nil;
+            [self clearNavigationBar];
         });
     }];
 }
 
+- (DMEScope *)createSampleScopeForOneYearOfSocialData
+{
+    // An example to create a scope for one last year of social data.
+    NSMutableArray *serviceObjectTypes = [NSMutableArray new];
+    for (int i = 1; i <= 60; i++)
+    {
+        DMEServiceObjectType *serviceObjectType = [[DMEServiceObjectType alloc]initWithIdentifier:i];
+        [serviceObjectTypes addObject:serviceObjectType];
+    }
+    
+    DMEServiceType *serviceTypeFacebook = [[DMEServiceType alloc] initWithIdentifier:1 objectTypes: serviceObjectTypes];
+    DMEServiceType *serviceTypeTwitter = [[DMEServiceType alloc] initWithIdentifier:3 objectTypes: serviceObjectTypes];
+    DMEServiceType *serviceTypeInstagram = [[DMEServiceType alloc] initWithIdentifier:4 objectTypes: serviceObjectTypes];
+    NSArray *serviceTypes = [NSArray arrayWithObjects: serviceTypeFacebook, serviceTypeTwitter, serviceTypeInstagram, nil];
+    DMEServiceGroup *serviceGroupSocial = [[DMEServiceGroup alloc] initWithIdentifier:1 serviceTypes:serviceTypes];
+
+    DMEScope *scope = [[DMEScope alloc] init];
+    scope.serviceGroups = [NSArray arrayWithObjects: serviceGroupSocial, nil];
+    
+    NSDate *to = [NSDate date];
+    NSUInteger componentFlags = NSCalendarUnitYear;
+    NSDateComponents *components = [[NSCalendar currentCalendar] components:componentFlags fromDate:to];
+    [components setYear:-1];
+    NSDate *from = [[NSCalendar currentCalendar] dateByAddingComponents:components toDate:to options:0];
+    
+    DMETimeRange *range = [DMETimeRange from:from to:to];
+    scope.timeRanges = [NSArray arrayWithObjects:range, nil];
+    return scope;
+}
+
+- (DMEPullConfiguration *)createSampleConfiguration
```

Comment:
One we reinstate the configuration section in `viewDidLoad` we can take this out as well.